src/common/Statistics.h

```diff
@@ -308,6 +315,12 @@ class Statistics
     */
     void printStartingIteration( unsigned long long iteration, String message );
 
+    /*
+      Print a long attribute in time format
+     */
+    void printLongAttributeAsTime( unsigned long long longAsNumber );
```

Comment:
Can this function be "const"?
Also, remove one of the two empty lines afterwards, keep just one